menderqa_pipeline.go

```diff
@@ -276,10 +278,18 @@ func getBuildParameters(log *logrus.Entry, conf *config, build *buildOptions) ([
 
 	// set the poky branch equal to the meta-mender base branch, unless it
 	// is master, in which case we rely on the default.
```

Comment:
This comment is a bit off now, it only applies for PRs on `meta-mender` repository.

Reply:
Indeed. Your comment also made me realize I had done it wrong. I was only setting poky, but I need to set meta-mender too. Check out the new one.